Mathlib/Data/PNat/Basic.lean

```diff
@@ -278,6 +278,13 @@ theorem pow_coe (m : ℕ+) (n : ℕ) : ↑(m ^ n) = (m : ℕ) ^ n :=
   rfl
 #align pnat.pow_coe PNat.pow_coe
 
+/-- b is greater one if any a is less than b -/
+theorem one_lt_of_lt {a b : ℕ+} (hab : a < b) : 1 < b := bot_le.trans_lt hab
+
+theorem add_one (a : ℕ+) : a + 1 = succPNat a := rfl
+
+theorem lt_succ_self (a : ℕ+) : a < succPNat a := lt.base a
```

Comment:
While you're at it, could you please add lemmas for `a < a + b` (`lt_add_right`) and `b < a + b` (`lt_add_left`) as well? The proofs can reuse lemmas about `Nat`. Thank you!

bors d+

Reply:
Will do later. I'm not done with `PNat`.

Reply:
@urkud They are already in PNat:
https://github.com/leanprover-community/mathlib4/blob/b119a280b5e1ea54d4b78fff2cedbbf5d3a4d81b/Mathlib/Data/PNat/Basic.lean#L251